### ISSUE TITLE
What do "offset" and "limit" do on the searchCount endpoint?

### ISSUE THREAD

**Issue by tcmitchell** [SynBioHub/api-docs#66]
Operating system: ✍️ TODO
Last upstream commit (run `git log --author="Robert Lord" | head -n 1`): ✍️ TODO
Browser version(s): ✍️ TODO
Ruby version (run `ruby -v`):  ✍️ TODO

---

https://synbiohub.github.io/api-docs/#count-search-results shows the following template for searchCount:

```
GET <SynBioHub URL>/searchCount/<key>=<value>&...&<search string>/?offset=#&limit=#
```

I'm surprised to see `offset` and `limit` there. What do those do on `searchCount`?

**Comment by jvscholz** [comment]
@cjmyers

**Comment by cjmyers** [comment]
You are correct.  Limit and offset should not be used for searchCount.  This is an error in the documentation.

**Comment by jvscholz** [comment]
fixed via #23bb9f0e5540806040cd63e3e659f3272520f114
